src/bundle/common.rs

```diff
@@ -39,7 +39,7 @@ fn symlink_dir(src: &Path, dst: &Path) -> io::Result<()> {
 }
 
 #[cfg(unix)]
-fn symlink_file(src: &Path, dst: &Path) -> io::Result<()> {
+pub fn symlink_file(src: &Path, dst: &Path) -> io::Result<()> {
```

Comment:
I'm not sure if there's another intended way to do this

Reply:
Not sure if I just missed another way of doing this, but AppDir specifies you can symlink the .desktop file and the icons to the AppDir root